onnxruntime/core/providers/webgpu/nn/layer_norm.h

```diff
@@ -11,25 +11,28 @@ namespace webgpu {
 
 class LayerNormProgram final : public Program<LayerNormProgram> {
  public:
-  LayerNormProgram(bool has_bias,
-                   bool is_fp16,
-                   bool simplified) : Program{"LayerNorm"},
-                                      has_bias_{has_bias},
-                                      is_fp16_{is_fp16},
-                                      simplified_{simplified} {}
+  LayerNormProgram(bool has_bias, bool is_fp16, bool simplified, bool has_mean_output,
+                   bool has_inv_std_dev_output)
+      : Program{"LayerNorm"},
+        has_bias_{has_bias},
+        is_fp16_{is_fp16},
+        simplified_{simplified},
+        has_mean_output_{has_mean_output},
+        has_inv_std_dev_output_{has_inv_std_dev_output} {}
 
   Status GenerateShaderCode(ShaderHelper& sh) const override;
 
-  WEBGPU_PROGRAM_DEFINE_UNIFORM_VARIABLES(
-      {"norm_count", ProgramUniformVariableDataType::Uint32},
-      {"norm_size", ProgramUniformVariableDataType::Uint32},
-      {"norm_size_vectorized", ProgramUniformVariableDataType::Uint32},
-      {"epsilon", ProgramUniformVariableDataType::Float32});
+  WEBGPU_PROGRAM_DEFINE_UNIFORM_VARIABLES({"norm_count", ProgramUniformVariableDataType::Uint32},
+                                          {"norm_size", ProgramUniformVariableDataType::Uint32},
+                                          {"norm_size_vectorized", ProgramUniformVariableDataType::Uint32},
+                                          {"epsilon", ProgramUniformVariableDataType::Float32});
 
  private:
   bool has_bias_;
   bool is_fp16_;
   bool simplified_;
+  bool has_mean_output_ = false;
+  bool has_inv_std_dev_output_ = false;
```

Comment:
Nit? "= false", not need to initialize here?

Reply:
Done!